reports/Update_Nov_29.md

```diff
@@ -0,0 +1,64 @@
+# Seenema [11/29/2023]
+
+## Team report (status update for your TA, including an agenda for the project meeting)
+The detail page format is complete. We added some color change to the buttons so users know that they have been pressed. 
+We also added some animations so that users can tell that there is action happening. We spent a few days working on trying
+to figure out how to get the search bar functionality to work they way we wanted. For example, where should you be taken if
+you hit the back button on the screen (not browser) if you have made a search? Back to the previous screen, which could be a 
+movie card, or back to the homepage? We completed the addition of the friend features and also have easy drop downs for 
+sending them a reccomendation.
+
+
+### Agenda for Project Meeting
+1. Demo of current functionality
+2. Discussing next steps - expectations and advice for the final write up and presentation
+3. Confirm any final edits to be made before due date (prioritize MVP functions)
+
+### Subsection 1:
+There might be design decisions left to make, but the biggest portion is completing the final presentation and write up.
+
+### Subsection 2:
+**Update (progress and issues):**
+No issues of note currently. 
+
+- **What we did:**
+    - Everyone took time off over the long weekend at various points
+    - Detail screen completed (with addition of 'add to friend's list'
+    - Search bar functional for search within a genre is completed 
+
+- **Insights:**
+No new insights this week. Just seeing the end approach and working to complete the project MVP by EOD 11/30 so we have
+time to dedicate to the final presentation and write up
+
+### Subsection 3:
+
+#### Back end goals for the upcoming week:
+- Verification
+
+#### Front end goals for the upcoming week:
+- User testing with friends/family
+
+**Contributions of individual team members:**
+
+##### Long Term Focus/Goal:
+- Working to keep up the positive momentum so that we are able to implement at least one of the stretch features
+
+**Anand:**
+- **Last week:** Search Bar integrated with Genre filtering
+- **This week:** Making sure navigating 'back' for users goes to the expected locations
+- **Next week:** Presentation and final write up
+
+**Danno:**
+- **Last week:** Finish UI fine tuning for Detail Page
+- **This week:** Testing to identify any bugs or unexpected behavior
+- **Next week:** Presentation and final write up
+
+**Dishant:**
+- **Last week:** Genre filtering completed
+- **This week:** Testing to identify any bugs or unexpected behavior
+- **Next week:** Presentation and final write up
+
+**Leena:**
+- **Last week:** Finalize the profile page UI
```

Comment:
Here last week, I worked on add friend, add movies to friend's suggestion list and add movies to my list features. I think you'll need to add that here.